TROUBLESHOOTING.md

```diff
@@ -0,0 +1,18 @@
+If you have a non-library SendGrid issue, please contact our [support team](https://support.sendgrid.com).
+
+If you can't find a solution below, please open an [issue](https://github.com/sendgrid/java-http-client/issues).
+
+## Table of Contents
+
+* [Viewing the Request Body](#request-body)
+
+<a name="request-body"></a>
+## Viewing the Request Body
+
+When debugging or testing, it may be useful to examine the raw request body to compare against the [documented format](https://sendgrid.com/docs/API_Reference/api_v3.html).
+
+You can do this right before you call `request.setBody(mail.build())` like so:
+
+```java
+System.out.println(mail.build());
```

Comment:
Would this be more direct for outputting the request body?
```
 System.out.println(request.getBody());
```

[I found this method here](https://github.com/sendgrid/java-http-client/blob/master/src/main/java/com/sendgrid/Request.java#L87)

Reply:
Makes sense, updated.